### ISSUE TITLE
Check logging configuration

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#602]
DSpace 6 dspace.cfg
```
##### Logging configuration #####
# Override default log4j configuration file
# You may provide your own configuration here, existing alternatives are:
# log.init.config = ${dspace.dir}/config/log4j-console.properties
log.init.config = ${dspace.dir}/config/log4j.properties

# All other log4j settings moved to config/log4j.properties

###### Statistical Report Configuration Settings ######

# should the stats be publicly available?  should be set to false if you only
# want administrators to access the stats, or you do not intend to generate
# any
report.public = false

# directory where live reports are stored
report.dir = ${dspace.dir}/reports/

# directory where logs are stored
log.report.dir = ${dspace.dir}/log
```
DSpace 6 log4j.properties differences from DSpace 5
```
###########################################################################
# log4j.properties
#
# This is the primary log4j (logging) configuration file for DSpace. By default,
# Log4j is configured to write log files that rotate daily. However, you may
# tweak these settings based on your local needs / best practices.
# For more information on log4j configuration, see:
# https://logging.apache.org/log4j/1.2/manual.html
###########################################################################

# VARIABLES:
# The following variables can be used to easily tweak the default log4j settings.
# These variables are used by the log4j config / appenders later in this file.

# log.dir
#   Default log file directory for DSpace. Defaults to the 'log' subdirectory
#   under [dspace.dir]. NOTE: The value of 'dspace.dir' will be replaced by
#   its value in your configuration when DSpace is deployed (via Ant).
log.dir=${dspace.dir}/log

# loglevel.dspace
#   Log level for all DSpace-specific code (org.dspace.*)
#   Possible values (from most to least info): DEBUG, INFO, WARN, ERROR, FATAL
#   Defaults to INFO
loglevel.dspace=INFO

# loglevel.other
#   Log level for other third-party tools/APIs used by DSpace
#   Possible values (from most to least info): DEBUG, INFO, WARN, ERROR, FATAL
#   Defaults to INFO
loglevel.other=INFO
```

DSpace 5 settings
```
##### Logging configuration #####

# Override default log4j configuration
# You may provide your own configuration here, existing alternatives are:
# log.init.config = ${dspace.dir}/config/log4j.xml
# log.init.config = ${dspace.dir}/config/log4j-console.properties
log.init.config = ${dspace.dir}/config/log4j.properties

# Where to put the logs (used in configuration only)
log.dir = ${dspace.dir}/log
```

**Comment by alawvt** [comment]
@keithgee, I'd appreciate any help on this. I created the branch, `add_media_filters_to_local.cfg` for Issue #596 and added the new media filter, mediafilter.PoiWordFilter. It worked, creating .txt files from a .pdf, .doc, and .docx. However the mediafilter commands failed to create thumbnails of these files. The errors included problems with the location and permissions of the log files. The log files should be in /dspace/log/ and owned by vtechworks. That directory exists but is empty. The directory /log/ contains these log files and they are owned by root instead of vtechworks.
```
-rw-r--r--  1 root root     0 Jul 10 19:01 checker.log.2019-07-10
-rw-r--r--  1 root root 85800 Jul 10 19:07 cocoon.log.2019-07-10
-rw-r--r--  1 root root 21382 Jul 10 19:07 dspace.log.2019-07-10
```
`sudo -u vtechworks ./dspace filter-media -i 10919/87721`
```
log4j:ERROR setFile(null,true) call failed.
java.io.FileNotFoundException: /log/dspace.log.2019-07-10 (Permission denied)
log4j:ERROR setFile(null,true) call failed.
java.io.FileNotFoundException: /log/checker.log.2019-07-10 (Permission denied)
log4j:ERROR setFile(null,true) call failed.
java.io.FileNotFoundException: /log/cocoon.log.2019-07-10 (Permission denied)
```

**Comment by keithgee** [comment]
@alawvt I'm looking at this now. Please join me in Slack if you are available.

**Comment by keithgee** [comment]
I've fixed this issue with commit 7258c17.

The dspace.dir parameter was no longer in the local.cfg file; I put it back in.

This doesn't really mesh with my understanding of how the configuration system works - I thought that if this variable was in dspace.cfg (it is), it would be available everywhere.

Closing.

**Comment by alawvt** [comment]
I just built this and the logs are back where they belong. Thank you very much @keithgee.
